docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Day 2 Operations with Keptn
+description: How to operate and maintain your Keptn Apps
+weight: 20
+---
+
+After you have successfully rolled out your application by following
+the instructions in the [integration guide](../integrate),
+Keptn also assists you with day 2 operations for your application.
+
+Tasks that fall under this category could be:
+
+* Updating the version of one or more workloads that are part of
+the same application.
+* Adding a new workload to an existing application
+* Monitoring the health of your application using `KeptnMetrics`
+* Optimizing the resource usage of your applications by integrating
+`KeptnMetrics` into a
+[HorizontalPodAutoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+## Updating Workload Versions
+
+After a first successful deployment of your application with Keptn,
+you will eventually have the need for updating the version of one or
+more workloads that are part of the application.
+This will usually involve updating the image of a deployment,
+but also a change in the configuration of a deployment
+(e.g. using a different service account name for a pod) could be seen as
+an update.
+Regardless of that however, it is the user who decides what Keptn
+will see as a version bump in the application by setting the value of
+the `keptn.sh/version` or `app.kubernetes.io/version` labels/annotations
+in their workloads.
+
+Only when this changes, Keptn will interpret a change as a new version
+and will thus re-run the pre- and post-tasks/evaluations for the application.
+
+If the version label/annotation does not change, Keptn will not consider
+a change of a workload configuration as an update, and therefore no pre-
+and post-tasks/evaluations will be executed as those have already been
+completed for the related `WorkloadVersion`.
```

Comment:
```suggestion
If the version label/annotation does not change, Keptn does not consider
a change of a workload configuration to be an update, and therefore no pre-
and post-tasks/evaluations are executed because they have already been
completed for the related `WorkloadVersion`.
```